site/layouts/_default/tutorial.html

```diff
@@ -41,6 +41,47 @@
           <div class="grid grid-cols-12">
             <!-- Column: Sidebar -->
             <div class="col-span-4 mb-5 hidden px-5 lg:block">
+              <a
+                id="newDC-tutorials"
+                href="https://twitter.com/share?url=Looking%20for%20a%20beginner%20%F0%9F%8E%93%20web%20development%20course%3F%20%20%0A%0AThis%20is%20my%20recommendation.%20%F0%9F%92%AA%0A%20%0AYou%20will%20learn%20%23Bootstrap%2C%20%23MaterialDesign%2C%20%23UI%2C%20%23UX%2C%20%23webdesign%20best%20practices%20%26%20much%20more.%20%0A%0ALearn%20directly%20from%20%40ascensus_mdb%20the%20founder%20%F0%9F%8C%B1%20of%20%40MDBootstrap.%0A%0Ahttps%3A%2F%2Ft.ly%2FpZyh"
+                target="_blank"
+                style="transition: opacity 500ms ease-in 0s; opacity: 1;">
```

Comment:
It should work with tailwind class:
`class="opacity-100 transition-opacity duration-500 ease-in`